docs/multi-cluster-mesh/active-passive/multi-cluster-active-passive.md

```diff
@@ -0,0 +1,106 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
+Setup three clusters (one active – two passive) in a single mesh multi network (two in the same network, one in a different network) setup with Istio operator.
+
+## Setup: 
+
+### Steps:
+
+#### Active setup:
+1. Install cluster registry controller:
+```
+helm install --namespace=cluster-registry --create-namespace cluster-registry-controller deploy/charts/cluster-registry  --set image.tag=v0.1.9 --set localCluster.name=demo-active --set network.name=network1
+```
+2. Install istio operator:
+```
+helm install --namespace=istio-system --create-namespace istio-operator-v113x deploy/charts/istio-operator --set clusterRegistry.clusterAPI.enabled=true --set clusterRegistry.resourceSyncRules.enabled=true --set image.tag=v2.13.2
+```
+3. Apply ACTIVE ICP CR:
+```
+kubectl -n=istio-system apply -f https://raw.githubusercontent.com/banzaicloud/istio-operator/9948582d6a0dd35af59739ca286a3e72c8ba810c/docs/multi-cluster-mesh/active-passive/active-icp.yaml
+```
+
+#### Passive-1 setup (different network):
+- Install cluster registry controller (replace Kubernetes API server endpoint):
+```
+helm install --namespace=cluster-registry --create-namespace cluster-registry-controller deploy/charts/cluster-registry  --set image.tag=v0.1.9 --set localCluster.name=demo-passive-1 --set network.name=network2 --set controller.apiServerEndpointAddress=<KUBERNETES_API_SERVER_FOR_ACTIVE>
```

Comment:
Mention why `apiServerEndpointAddress` might need to be configured, you can refer to cluster registry doc.